### PR TITLE
Feature/events

### PR DESCRIPTION
**Pull request by Braingenius2**
#  Implemented Event Handling and Math Logic in Calculator Component
In this pull request, I implemented the following

- [x] created a new directory called logic and copied the files calculate.js and operate.js into it.
- [x] installed the big.js library using NPM.
- [x] analyzed and improved on the calculate.js and operate.js files.
- [x] imported the calculate.js file in the Calculator.js component.
- [x] implemented click event handlers in the Calculator component using the math logic from calculate.js and operate.js.
- [x] tested the application in the browser to ensure the Calculator performs math operations correctly.

### CODE REVIEW

**Comment by codecaiine** [comment]
Hi @Braingenius2  The link to your issue is broken can you fix it ?

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,44 +1,56 @@
...
       </div>
-      <div className="row">
-        <Button className="number zero" label="0" />
-        <Button label="." />
-        <Button className="operation" label="=" />
+      <div className="buttons">
+        <div className="row">
+          <Button onClick={() => handleClick('AC')} label="AC" />
+          <Button onClick={() => handleClick('+/-')} label="+/-" />
+          <Button onClick={() => handleClick('%')} label="%" />
```
Comment (1GORDON) [inline src/components/Calculator.js:25]: Great job @Braingenius2 🤝

- [ ] Your modulus button is not working as intended. Make sure it returns a remainder or 0. Kindly fix this and your project will be ready for approval.

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Braingenius2 ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @Braingenius2, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Good Highlights✔️

- Module operation now works as expected

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**